apps/studio/components/interfaces/Auth/Users/UsersFooter.tsx

```diff
@@ -42,6 +43,13 @@ export const UsersFooter = ({
       filter: filter === 'all' ? undefined : filter,
       providers: selectedProviders,
       forceExactCount,
+      // Force exact count for optimized search since result sets are small
+      // forceExactCount: forceExactCount || specificFilterColumn !== 'freeform',
```

Comment:
I wasn't super sure if we always want a specific count when the filter is applied. This is kinda the big reason why all the estimates are so strange. Right now the count query users an explain, inside the `count_estimate` function so its counting the stats row? We can remove this but wanted to ask if we should use the exact value here now that we have proper count filtering

Reply:
hmm from a performance POV i think we should still default to false for this unless unified search
so that we default to checking rel_tuples initially rather than running `count(*)`

> estimates are so strange

this is alright imo - it's expected that the estimates are not always accurate and there's an escape hatch to fetch the actual row count which we show a warning before we run the query

altho please do let me know if i might be overlooking anything! 🙏